src/api/app/assets/stylesheets/webui2/modals.scss

```diff
@@ -5,3 +5,24 @@
 .modal-body p {
     white-space: pre-wrap;
 }
+
+.modal-footer {
+  a {
+    &.btn-outline-danger {
+      @extend .text-danger;
+      &:hover { color: $white !important; }
```

Comment:
I don't get why we have to do this. Why does this work in the [bootstrap docu](https://getbootstrap.com/docs/4.1/components/buttons/#outline-buttons) and not here?

Reply:
They are links that look like buttons, not real buttons. And these links inherit black color for text.

Even in the bootstrap page, if we open the inspector and replace the `button` HTML tag for an `a` tag, it shows black text.

**As button:**
![screenshot-2018-10-2 buttons](https://user-images.githubusercontent.com/2581944/46353088-a0fc1a00-c65b-11e8-8017-432a8b57bfde.png)

**As anchor:**
![screenshot-2018-10-2 buttons 1](https://user-images.githubusercontent.com/2581944/46353131-ba04cb00-c65b-11e8-8d8b-521d2ca05d9e.png)

Reply:
you can apply the bootstrapp btn classes also to the `<a>` element...

Reply:
As you can see [here](https://github.com/openSUSE/open-build-service/blob/53c1fc22ee2e3c2a81694e99f4b32ced953b1a75/src/api/app/views/webui2/webui/package/_delete_dialog.html.haml#L27), they already have the `.btn` class:

`%a.btn.btn-sm.btn-outline-secondary.px-4{ data: { dismiss: 'modal' } }`

I'm going to rethink about it and try to find a better solution!